pkg/topology/topology.go

```diff
@@ -106,8 +106,16 @@ func getTopologyHint(sysFSPath string) (*Hint, error) {
 
 	hint := Hint{Provider: sysFSPath}
 	fileMap := map[string]*string{
-		"local_cpulist": &hint.CPUs,
-		"numa_node":     &hint.NUMAs,
+		// match /sys/devices/system/node/node0/cpulist
+		"cpulist":         &hint.CPUs,
+		// match /sys/devices/pci0000:00/pci_bus/0000:00/cpulistaffinity
+		"cpulistaffinity": &hint.CPUs,
```

Comment:
I'm not 100% sure about this one. In subfunction entries of PCI devices, the topology func should go up one layer in tree via `device` and should see `local_cpulist` entry. Have you seen scenarios where this would be more reliable compared to root function of the pci device?